pom.xml

```
@@ -12,7 +12,12 @@
         <repository>
             <id>jenkins-releases</id>
             <name>Jenkins Releases</name>
-            <url>http://repo.jenkins-ci.org/releases</url>
+            <url>https://repo.jenkins-ci.org/releases</url>
+        </repository>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
```

Just a nitpick :) I would suggest to use the same form of the `<id>` as in `jenkins-releases`:
```suggestion
            <id>jenkins-public</id>
```

---

In fact this is to correct the URL of a dependency repository definition, I will add a comment

---

added the comment